specification/resources/resource-manager/Microsoft.Solutions/stable/2019-03-01/managedapplications.json

```diff
@@ -0,0 +1,1889 @@
+            "description": "The name of the JIT request."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },          
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "NoContent"
+          },
+          "200": {
+            "description": "OK"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Solutions/jitRequests": {
+      "get": {
+        "tags": [
+          "jitRequests"
+        ],
+        "operationId": "jitRequests_ListBySubscription",
+        "description": "Retrieves all JIT requests within the subscription.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },          
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of JIT requests.",
+            "schema": {
+              "$ref": "#/definitions/JitRequestDefinitionListResult"
+            }
+          },         
+          "404": {
+            "description": "Not Found"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        } 
+      }     
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Solutions/jitRequests": {
+      "get": {
+        "get": {
+          "tags": [
+            "jitRequests"
+          ],
+          "operationId": "jitRequests_ListByResourceGroup",
+          "description": "Retrieves all JIT requests within the subscription.",
+          "parameters": [
+            {
+              "name": "resourceGroupName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The name of the resource group. The name is case insensitive.",
+              "pattern": "^[-\\w\\._\\(\\)]+$",
+              "minLength": 1,
+              "maxLength": 90
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            },          
+            {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK - Returns an array of JIT requests.",
+              "schema": {
+                "$ref": "#/definitions/JitRequestDefinitionListResult"
+              }
+            },         
+            "404": {
```

Comment:
This indicates that a 404 should be treated as a success, is that intentional?  ARM feedback will be interesting here.

Reply:
I was following the convention of previous versions and other ARM APIs. Is there a general guideline for 404?

Reply:
I've not seen this before so it was a surprise to me, doesn't necessarily mean it's incorrect.  The ARM reviewer can provide a definitive answer.

Reply:
Do you know who's the ARM reviewer? If it's from ARM team, I could talk to them offline.

Reply:
Looks like one hasn't been assigned yet.  If this is urgent you can email the ARM on-call person (there are instructions on how to find this info in the ARM API Review Checklist comment at the top.